### ISSUE TITLE
Missing label on "username or email" field on manager login screen (accessibility defect)

### ISSUE THREAD

**Issue by paulbohman** [modxcms/revolution#12144]
For compliance with Web Content Accessibility Guidelines (WCAG)

The "username or email" field needs a <label> tag. Placeholder text is not considered a valid label, in part because it disappears on focus or when user starts typing.

See accessibility to-do list at https://dequeuniversity.com/modx-accessibility.html#login

**Comment by rthrash** [comment]
This is being addressed at the [MODX Accessible CMS Project](https://github.com/modxcms/a11y/issues/), and with Paul's agreement has been moved there. For more details, see https://github.com/modxcms/revolution/issues/12156#issuecomment-72382480. Thank you!
